ui/imports/shared/controls/AddressSourceSelector.qml

```diff
@@ -40,9 +41,9 @@ Item {
         selectMenu.delegate: StatusMenuItemDelegate {
             statusPopupMenu: select
             action: StatusMenuItem {
-                text: root.sources[index].text
+                text: root.sources[currentIndex].text
```

Comment:
Uhm.. I think this is gonna error when `currentIndex == -1`.

Any reason we a) switched to `currentIndex` from `index` and b) this was initialized with `-1`?

Reply:
It looks like this is still here?

Reply:
right, I missed that. Removed and rebased!